payment-initiation-api/src/main/java/com/aevi/sdk/pos/flow/model/PaymentFlowConfiguration.java

```diff
@@ -0,0 +1,188 @@
+/*
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.aevi.sdk.pos.flow.model;
+
+import com.aevi.sdk.flow.model.config.AppExecutionType;
+import com.aevi.sdk.flow.model.config.FlowApp;
+import com.aevi.sdk.flow.model.config.FlowConfig;
+import com.aevi.sdk.flow.model.config.FlowStage;
+import com.aevi.util.json.JsonConverter;
+import com.aevi.util.json.Jsonable;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Represents the payment flow configuration.
+ */
+public class PaymentFlowConfiguration implements Jsonable {
+
+    private final List<FlowConfig> flowConfigurations;
+    private final PaymentFlowServices allServices;
+    private final List<String> flowNames;
+    private final Set<String> flowTypes;
+
+    public PaymentFlowConfiguration(List<FlowConfig> flowConfigurations, PaymentFlowServices paymentFlowServices, String callingPackageId) {
+        this.flowConfigurations = flowConfigurations;
```

Comment:
This looks like it is keeping the full list of `flowConfigurations` here instead of filtering them down as per the `flowNames` and `flowTypes` below

Reply:
Yes, in fact the filtering should be moved to FPS side instead.